File: mvn-advent-of-code/src/test/java/advent/twenty_eighteen/Day06ChronalCoordinatesTest.java
package advent.twenty_eighteen;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day06ChronalCoordinatesTest {
    @Test
    public void checkReal() {
        Day06ChronalCoordinates cc = new Day06ChronalCoordinates("/twenty_eighteen/Day06-ChronalCoordinates-input.txt", 10000);

        assertEquals(5975, cc.getPart1Answer().intValue());
        assertEquals(38670, cc.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        Day06ChronalCoordinates cc = new Day06ChronalCoordinates("/twenty_eighteen/Day06-ChronalCoordinates-example#1.txt", 32);

        assertEquals(17, cc.getPart1Answer().intValue());
        assertEquals(16, cc.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_eighteen/Day06ChronalCoordinatesTest.scala
package advent.twenty_eighteen

import org.scalatest.{FlatSpec, Matchers}

class Day06ChronalCoordinatesTest  extends FlatSpec with Matchers {
  "2018 Day 06 - Input" should "calculate answers" in {
    val dp = new Day06ChronalCoordinates("/twenty_eighteen/Day06-ChronalCoordinates-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day06ChronalCoordinates("/twenty_eighteen/Day06-ChronalCoordinates-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
